common/src/main/java/org/terracotta/angela/common/tcconfig/TsaConfig.java

```diff
@@ -47,7 +47,7 @@ public static TsaConfig tsaConfig(Version version, List<TsaStripeConfig> stripeC
   }
 
   public static TsaConfig tsaConfig(Version version, List<TsaStripeConfig> stripeConfigs, Supplier<TcConfig> tcConfigSupplier) {
-    if (version.getMajor() < 10) {
```

Comment:
@aurbroszniowski : I switch to `== 4`, like every other places in the code base. I don't know why `< 10` was used here (and only here).

Reply:
And I don't even know why this limitation is here because the TcConfig type claims to support 4.x and 10.x. Maybe there's a alternate method.